tests/test_star.py

```diff
@@ -132,6 +132,30 @@ def test_apply_many(self):
         self.assertFloatsAlmostEqual(ras, expect.ra.to_value(u.degree), rtol=3e-7)
         self.assertFloatsAlmostEqual(decs, expect.dec.to_value(u.degree), rtol=6e-7)
 
+    def test_apply_many_reverse(self):
+        """Test apply for a negative time shift over a range of points on the
+        sphere."""
+        expect = self.coords.apply_space_motion(dt=-self.dt)
+
+        ras = np.zeros(len(expect))
+        decs = np.zeros(len(expect))
+        for i, x in enumerate(self.coords):
+            self.coords
+            star = lsst.jointcal.BaseStar(x.ra.value, x.dec.value, 100, 100*0.001)
```

Comment:
It's probably not worth it given jointcal's pending deprecation, but the internals of this loop are almost identical in this and the above test, so they'd be better as a free or static function to share. I don't think it's necessary on this ticket though.